apps/ledger-live-desktop/src/renderer/reducers/settings.ts

```diff
@@ -460,22 +469,23 @@ const defaultsForCurrency: (a: Currency) => CurrencySettings = crypto => {
   };
 };
 
-export type SupportedCoutervaluesData = {
+export type supportedCountervaluesData = {
   value: string;
   label: string;
   currency: Currency;
 };
-export const supportedCountervalues: SupportedCoutervaluesData[] = [
-  ...listSupportedFiats(),
-  ...possibleIntermediaries,
-]
-  .map(currency => ({
-    value: currency.ticker,
-    label: `${currency.name} - ${currency.ticker}`,
-    currency,
-  }))
-  .sort((a, b) => (a.currency.name < b.currency.name ? -1 : 1));
 
+export const getsupportedCountervalues = async (): Promise<supportedCountervaluesData[]> => {
```

Comment:
typo `s` -> `S` on the get**s**upportedCountervalues function and **s**upportedCountervaluesData type